addons/mail/static/src/discuss/core/common/discuss_channel_model.js

```diff
@@ -64,6 +67,13 @@ export class DiscussChannel extends Record {
         onDelete: (r) => r?.delete(),
     });
 
+    delete(options = { closeChatWindow: true }) {
+        if (this.chatWindow?.exists() && options.closeChatWindow) {
+            this.chatWindow.close();
+        }
+        super.delete(...arguments);
+    }
```

Comment:
delete is a low level method of records, it doesn't (and shouldn't) take a parameter.

Business code giving `closeChatWindow: false` here should be adapted to properly handle the situation instead.
It makes no sense to keep a chat window if the channel is deleted.

If I'm seeing the issue correctly, we would want to call delete of the channel after the super of close in the live chat override?

Reply:
> If I'm seeing the issue correctly, we would want to call delete of the channel after the super of close in the live chat override?

I tried but that did work, you get an infinity loop of close/delete. I didn't find a better solution than this ugly param.